test/IntegrationTests/Microsoft.Testing.Platform.Acceptance.IntegrationTests/HelpInfoTests.cs

```diff
@@ -49,6 +49,38 @@ public async Task Help_WhenNoExtensionRegistered_OutputDefaultHelpContent(string
         testHostResult.AssertOutputMatches(wildcardMatchPattern);
     }
 
+    [ArgumentsProvider(nameof(TargetFrameworks.All), typeof(TargetFrameworks))]
+    public async Task HelpShortName1_WhenNoExtensionRegistered_OutputDefaultHelpContent(string tfm)
+    {
+        var testHost = TestInfrastructure.TestHost.LocateFrom(_testAssetFixture.NoExtensionTargetAssetPath, TestAssetFixture.NoExtensionAssetName, tfm);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--h");
```

Comment:
Does `-h` work?  Because I would never think about using `--h` and I guess we are going for backward compatibility?